### ISSUE TITLE
Repentogon's completion marks are overlapped by these ones

### ISSUE THREAD

**Issue by calvinhobbes1010** [ConnorForan/PauseScreenCompletionMarksAPI#3]
Whenever I play as a custom character that uses this API, while I also have Repentogon installed, both of their post-it notes end up overlapping each other, making it difficult to read.

(Using [The Sheriff mod](https://steamcommunity.com/sharedfiles/filedetails/?id=3082060097) as an example)

![20240720005331_1](https://github.com/user-attachments/assets/94a8bda5-f2ae-4c56-883f-58bd55f9292b)

**Comment by calvinhobbes1010** [comment]
And I experimented a bit on my end, and it seems like simply adding a Repentogon check in the `EntryPoint` and/or `Shaderhook` functions might do the trick.

**Comment by ConnorForan** [comment]
The overlapping is mainly this egregious for minimal resprite huds like this one, with the normal hud its not nearly as bad.

I'm unsure if I want to do anything on this side of things. Mods don't need to use this for their character's completion marks if repentogon is enabled and they are syncing/tracking their marks through repentogon, which is easy enough to check for. For old mods that use this that have not updated to use repentogon, that also means that repentogon likely does not have their previously-earned marks tracked, so it feels better to have it continue to render the marks that the mod itself has saved, over the default post it note (though yeah it looks bad for this custom hud specifically, mainly due to the transparency which is hard for me to account for).

Additionally, this library does have an additional (rarely-used) feature to render a custom post-it note for custom marks, which should still get rendered even with repentogon.

So in summary I feel that this is something that mods should handle in their transition/support for repentogon. For mods that used this library but have never updated with intentional repentogon support, should probably continue trying to render the marks they want to render. If they update their mods to sync their marks with repentogon then they don't need this anymore.

Though at the very least I can let the sheriff dev know about this since it is an easy fix.

**Comment by ConnorForan** [comment]
Actually, it looks like sheriff does not even load the PauseScreenCompletionMarksAPI if REPENTOGON is enabled...?

Do you have this installed: https://steamcommunity.com/sharedfiles/filedetails/?id=2503022727

**Comment by calvinhobbes1010** [comment]
Yeah, the sheriff dev tried fixing this issue on their end, but I'm still running into it.
I have the completion marks mod installed, but it's been disabled for a while now.
The sheriff mod also uses the [Unlock API](https://github.com/dsju/unlockapi) to my knowledge, and that library seems to use the PauseScreenCompletionMarksAPI as well in some capacity.
